src/pipx/commands/pin.py

```diff
@@ -0,0 +1,34 @@
+import logging
+from pathlib import Path
+
+from pipx.constants import ExitCode
+from pipx.emojis import hazard
+from pipx.util import PipxError
+from pipx.venv import Venv
+
+logger = logging.getLogger(__name__)
+
+
+def pin(venv_dir: Path, verbose: bool, unpin: bool) -> ExitCode:
```

Comment:
To comply with the function name, we might use a default value of `False` for `unpin`?